pkg/plugin/plugin.go

```diff
@@ -189,24 +191,15 @@ func getBridgeName(bridgeName, ovnPort string) (string, error) {
 	return "", fmt.Errorf("failed to get bridge name")
 }
 
-func attachIfaceToBridge(hostIfaceName string, contIfaceName string, brName string, vlanTag *uint, contNetnsPath string, ovnPortName string) error {
-	// Set external IDs so we are able to find and remove the port from OVS
-	// database when CNI DEL is called.
-	command := []string{
-		"--", "add-port", brName, hostIfaceName,
-		"--", "set", "Port", hostIfaceName, fmt.Sprintf("external-ids:contNetns=%s", contNetnsPath),
-		"--", "set", "Port", hostIfaceName, fmt.Sprintf("external-ids:contIface=%s", contIfaceName),
-	}
+func attachIfaceToBridge(ovsDriver *ovsdb.OvsBridgeDriver, hostIfaceName string, contIfaceName string, vlanTag *uint, contNetnsPath string, ovnPortName string) error {
+	var vlanTagNum uint = 0
```

Comment:
Let's make it `int` and use -1 to enable trunk. I vaguely remember VLAN 0 had some special significance.

Reply:
I think this change need to go to another PR is not related to this one.

we need to change the netConf struct
```
type netConf struct {
	types.NetConf
	BrName  string `json:"bridge,omitempty"`
	VlanTag *uint  `json:"vlan"`
}
```

and also this mean we need to update the documents and tell the user to config -1 if he want to use trunk (not so nice in my opinion)

Reply:
Ah, darn, sure, get it now.